File: src/buldringno/Entities/Boulder.cs
using System;
using System.Collections.Generic;

namespace BuldringNo.Entities
{
    public class Boulder : IEntityBase
    {
        public Boulder()
        {
            Problems = new List<Problem>(); 
        }
        public int Id { get; set; }
        public string Title { get; set; }
        public string DescriptionMain { get; set; }
        public string DescriptionSecondary { get; set; }
        public string Size { get; set; }
        public string Return { get; set; }
        public double GPSNorth { get; set; }
        public double GPSSouth { get; set; }
        public virtual Area Area { get; set; }
        public int AreaId { get; set; }
        public DateTime DateCreated { get; set; }
        public virtual ICollection<Problem> Problems { get; set; }
    }
}

File: src/buldringno/wwwroot/lib/spa/core/domain/boulder.js
"use strict";
var Boulder = (function () {
    function Boulder(id, title, descriptionMain, descriptionSecondary, thumbnail, areaId, areaTitle, totalProblems, dateCreated) {
        this.Id = id;
        this.Title = title;
        this.DescriptionMain = descriptionMain;
        this.DescriptionSecondary = descriptionSecondary;
        this.Thumbnail = thumbnail;
        this.AreaId = areaId;
        this.AreaTitle = areaTitle;
        this.TotalProblems = totalProblems;
        this.DateCreated = dateCreated;
    }
    return Boulder;
}());
exports.Boulder = Boulder;
